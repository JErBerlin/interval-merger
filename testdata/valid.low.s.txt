[1,3]  [4,5]  [8,10] [2,4] [11,12] [7,9]  [0,1] 
