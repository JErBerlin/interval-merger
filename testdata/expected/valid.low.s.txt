[0,5] [7,10] [11,12]
